apps/emqx_bridge/src/schema/emqx_bridge_v2_schema.erl

```diff
@@ -231,6 +251,23 @@ bridge_api_union(Refs) ->
             end
     end.
 
+unknown_bridge_schema(BridgeV2Type) ->
+    erroneous_value_schema(BridgeV2Type, <<"unknown bridge type">>).
+
+unknown_source_schema(SourceType) ->
+    erroneous_value_schema(SourceType, <<"unknown source type">>).
+
+-dialyzer({nowarn_function, [erroneous_value_schema/2]}).
```

Comment:
maybe add a comment here to describe why nowan_function.

Reply:
does writing a spec like below help? 
`-spec erroneous_value_schema(_Value, _Reason) -> no_return().`

Reply:
it does the trick for me in other cases,
however it's not exactly this function which does not return, so it might be different.
sorry for the noise

Reply:
>  however it's not exactly this function which does not return, so it might be different.

Yeah I tried this first but dialyzer still wasn't happy. 🫠